e2e-tests/playwright/e2e/github-happy-path.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { test, expect, Page } from '@playwright/test';
+import { UIhelper } from '../utils/UIhelper';
+import { Common } from '../utils/Common';
+import {
+  BackstageShowcase,
+  CatalogImport,
+} from '../support/pages/CatalogImport';
+
+test.describe('GitHub Happy path', () => {
+  let page: Page;
+  let common: Common;
+  let uiHelper: UIhelper;
+  let catalogImport: CatalogImport;
+  let backstageShowcase: BackstageShowcase;
+
+  const component =
+    'https://github.com/janus-idp/backstage-showcase/blob/main/catalog-entities/all.yaml';
+
+  test.beforeEach(async ({ browser }) => {
```

Comment:
Same comment that @subhashkhileri Subashi added on the other review comment. 

Can we use the beforeAll hook instead of beforeEach to avoid logging in for every test?

Reply:
There's a good reasons to use beforeEach instead of beforeAll while using Playwright (this is different from Cypress).
But for Github tests, it is not enabled, it will be create a Jira specific to enable this tests and make is stable